.changeset/lovely-coins-give.md

```diff
@@ -3,3 +3,5 @@
 ---
 
 Deprecate and replace Well `variant` prop with `color` and update the background and border color to fallback to `white` instead of the `default` variant's grey.
+
+- Adds codemod and bin script to run from directs, ie: `../components/bin/codemod.sh src migrateWellVariantToColor.ts`
```

Comment:
Not really sure what this sentence means.

Also worth adding a README with a few examples :)